bundles/net.tourbook/src/net/tourbook/ui/tourChart/SlideoutTourChartOptions.java

```diff
@@ -101,6 +102,8 @@
    private Button    _chkShowValuePointTooltip;
    private Button    _chkSelectAllTimeSlices;
 
+   private Scale     _scaleNightSectionsOpacity;
```

Comment:
>The night section should be optional that the user can select/deselect it with a checkbox

is it really necessary since the user can just set the transparency to 0 and it will not be displayed ?

Reply:
> > The night section should be optional that the user can select/deselect it with a checkbox
> 
> is it really necessary since the user can just set the transparency to 0 and it will not be displayed ?

This is similar to the tour track opacity, with one click the user can enable/disable a feature and keeps the opacity value

The wording should be written with lowercase "&Night sections opacity", I know there are still some "wrong" wordings with uppercases

![mt opacity](https://user-images.githubusercontent.com/1283445/113812111-cbb81380-976d-11eb-9996-80a644953c24.png)

Reply:
Ok, will do that

Reply:
Fixed in next PR

Reply:
The night section part should be placed outside of the `Graph` group, it is similar like the SRTM data, so a good place would be somewhere around the SRTM data

![mt night section](https://user-images.githubusercontent.com/1283445/114731121-c5afcd00-9d41-11eb-8a5d-48f14a2c86cc.png)

Reply:
Fixed
![image](https://user-images.githubusercontent.com/25235665/114780241-a24b4900-9d66-11eb-83bb-66a76440a8ae.png)

Reply:
While I was at it, notice that I fixed several labels that were using casing for each word : "**S**how **V**alue **P**oint **T**ooltip" ...

Reply:
I've used uppercase propably for the same reason why you have used uppercase for `Tour Chart`.
`Value Point Tooltip` is a feature/tool in MT but I know it is not consistenlty written in all cases. At least on the websites I try to write a feature/tool with italic font.

![mt value point tooltip case](https://user-images.githubusercontent.com/1283445/114876583-c22e4b80-9dfe-11eb-88e9-a921fa88c587.png)

Reply:
Ok, I have reverted to Show Value Point &Tooltip